docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -5,57 +5,71 @@ permalink: /documentation/using-testcafe/common-concepts/browser-support.html
-Safari            | 6+     | `safari`
+Mozilla Firefox   | latest | `firefox`
+Opera             | latest | `opera`
+Safari            | latest | `safari`
 
-## Browser Provider Plugins
+TestCafe also supports the two latest versions of Android and iOS browsers.
 
-If you need to use a browser that is not [supported directly](#directly-supported-browsers), create a [browser provider plugin](../../extending-testcafe/browser-provider-plugin/).
+You can specify local browsers by using *browser aliases* (see the table above) or paths to their executable files.
+The list of all the available browsers can be obtained by calling the [--list-browsers](../command-line-interface.md#-b---list-browsers) command.
 
-Created provider packages can be published to npm. The provider package name consists of two parts - the `testcafe-browser-provider-` prefix and the name of a provider itself; e.g., `testcafe-browser-provider-saucelabs`.
-So, you can search for available browser provider packages on npm by the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+## Portable Browsers
 
-For example, there are provider packages for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
+To use portable browsers, specify them by paths to browser executable files.
 
-* [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
-* [testcafe-browser-provider-phantomjs](https://www.npmjs.com/package/testcafe-browser-provider-phantomjs)
+## Remote Browsers
 
-You can install browser provider packages from npm as you would install any other plugin. See [Installing Plugins](../../extending-testcafe/index.md#installing-plugins).
+TestCafe can run tests in browsers on remote devices. The only requirement is a device should have network access to the TestCafe server.
 
-## Specifying Browsers for Test Task
+At first, you need to create a remote browser connection. You can do this from CLI by specifying the `remote` *alias* or from API by using the [createBrowserConnection](../programming-interface/testcafe.html#createbrowserconnection) method.
 
-If you are working with TestCafe through the command line,
-[list the desired browsers](../command-line-interface.md#browser-list)
-within the `testcafe` command.
+After that, TestCafe will provide URLs that you need to open in required browsers on your remote device to start testing.
 
-In case you are using the API, specify the browsers
-with the [runner.browsers](../programming-interface/runner.md#browsers) method.
+## Cloud Browsers
+
+TestCafe allows you to use cloud browsers. These browsers can be accessed through *browser provider plugins*. For example, there is the [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs) plugin for testing in [Sauce Labs](https://saucelabs.com/).
```

Comment:
Can I have a link to the page that describes what the heck is *browser provider plugins*, please?